packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -114,16 +116,32 @@ const FlipbookControls = ({
   }
 
   const resetTimeout = () => {
+    setIterationCounter(0)
     if (timeoutRef.current) {
       clearInterval(timeoutRef.current)
     }
   }
 
+  const handleIterationCounter = () => {
+    if (playIterations !== Infinity) {
+      setIterationCounter(iterationCounter + 1)
+    }
+  }
+
   useEffect(() => {
     if (playing) {
-      timeoutRef.current = setTimeout(() => {
-        handleNext()
-      }, 500 / playbackSpeed)
+      if (playIterations === Infinity) {
+        timeoutRef.current = setTimeout(() => {
+          handleNext()
+        }, 500 / playbackSpeed)
```

Comment:
I think you can get rid of this block, because `iterationCounter < playIterations * locations.length` should always be true when `playIterations` is `Infinity`.

Reply:
Good point!